cafe/app/models/cart.rb

```diff
@@ -0,0 +1,40 @@
+class Cart < ActiveRecord::Base
+	belongs_to :user
+	has_many :menu_items, through: :order_items
+	has_many :order_items, dependent: :destroy
+
+
+	def sub_total
+		sum = 0
+		self.order_items.each do |item|
```

Comment:
order and cart doesn't have any relationship. We need to refine our model. We can discuss this.